buku

```diff
@@ -618,11 +619,15 @@ class BukuDb:
             return
 
         if not self.json:
-            prompt(results, self.noninteractive)
+            if delete is False:
+                prompt_open(results, self.noninteractive)
```

Comment:
I would prefer a single function but how about this?

Instead of touching several functions what if we provide a `d` keyword at prompt which would confirm and delete all the search results after sorting them by ID? I believe we show the prompt for search results only.